src/components/icons/_icons.scss

```diff
@@ -1,6 +1,92 @@
 $includeHtml: false !default;
 
+// Icon sizes :(
```

Comment:
Have you checked whether some of these variations could be replaced with another? (reduce them to minimal amount)